modules/apps/27-interchain-accounts/controller/keeper/handshake.go

```diff
@@ -67,8 +67,8 @@ func (k Keeper) OnChanOpenInit(
 			panic(fmt.Errorf("active channel mapping set for %s but channel does not exist in channel store", activeChannelID))
 		}
 
-		if channel.State == channeltypes.OPEN {
-			return "", errorsmod.Wrapf(icatypes.ErrActiveChannelAlreadySet, "existing active channel %s for portID %s is already OPEN", activeChannelID, portID)
+		if channel.State != channeltypes.CLOSED {
```

Comment:
This is needed on the host side as well?

Reply:
yeah I think the host side still uses the `channel.IsOpen()` func

Reply:
Yes, [it does](https://github.com/cosmos/ibc-go/blob/a7f85d5e037da02428f23c493b4584b088ef2c02/modules/apps/27-interchain-accounts/host/keeper/handshake.go#L57-L59).

Reply:
Opened https://github.com/cosmos/ibc-go/issues/5629.